src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2112,24 +2112,24 @@
           }
         },
 
-        "id": {
+        "automationDetails": {
           "description": "Automation details that describe this run.",
           "$ref": "#/definitions/runAutomationDetails"
         },
 
-        "aggregateIds": {
-          "description": "Automation details that describe the aggregate of runs to which this run belongs.",
-          "type": "array",
-          "minItems": 0,
-          "uniqueItems": true,
-          "default": [],
-          "items": {
-            "$ref": "#/definitions/runAutomationDetails"
-          }
+        "RunAggregates": {
```

Comment:
>RunAggregates [](start = 9, length = 13)

Yes, sorry, Harleen and I discussed this. I felt that if "id" was wrong then so was "aggregateIds" and I suggested "runAggregates" meaning the set of aggregates to which this run belongs. Having said that, I never made that change in the "guid" change draft. I'll go with whatever you want here, up to and including leaving "aggregateIds" alone.

Reply:
It's fine. I updated #341 with this suggestion, let's just make sure the text is accurate when it comes to it. :)

---
In reply to: [268453384](https://github.com/Microsoft/sarif-sdk/pull/1360#discussion_r268453384) [](ancestors = 268453384)